index.js

```diff
@@ -2,6 +2,21 @@ const TestRunner = require('./src/TestRunner')
 
 exports.runTests = async function (options) {
   return new Promise((resolve, reject) => {
+    // Look for the config file. If one isn't found, keep
+    // going up the directory tree until you're at root
+    while (!process.cwd === '/') {
+      try {
+        const configPath = process.cwd() + '.mocha-puppeteer-config'
+        const { mochaOptions, puppeteerOptions } = require(`./${configPath}`)
+        options = Object.assign(options, {
```

Comment:
As I was stating above, technically this project could be used via API instead of CLI. I think that if `options.mochaOptions` or `options.puppeteerOptions` is already provided, you should not `Object.assign` the respective option.

Perhaps if even one of those is provided, we don't even both trying to find the config file. Something to consider.

Reply:
Perhaps it might be better to treat whatever is loaded in from the `.mocha-puppeteer-config` file as the base config and allow options passed into `runTests` to work as overrides? Of course, there would need to be some clear logging to indicate this behavior.